### PR TITLE
Pre-deployment

### LINKED ISSUE #23: Add link to this repository
I think it would be interesting to have a link to the repository, so when you start showing this people can see all the effort and knowledge that is behind it ^^

Probably in "About" to the repository on in "Contact" to your github profile :thinking:

### PR DESCRIPTION
**Pull request by espimadrazo**
Changed relative paths to absolute and moved style.css to Styles folder

### CODE REVIEW

File: subpages/about/index.html [hunk 1]
```diff
@@ -23,46 +23,46 @@
...
-          <li><a href="../base-rate-fallacy/index.html">Base Rate Fallacy</a></li>
+          <li><a href="/subpages/self-fulfilling-prophesy/">Self Fulfilling Prophecy</a></li>
+          <li><a href="/subpages/impostor-syndrome/index.html">Impostor Syndrome</a></li>
+          <li><a href="/subpages/fermi-paradox/index.html">Fermi Paradox</a></li>
+          <li><a href="/subpages/phenomenon-of-baader-meinhof/index.html">Phenomenon of Baader-Meinhof</a></li>
+          <li><a href="/subpages/curse-of-knowledge/index.html">Curse of Knowledge</a></li>
+          <li><a href="/subpages/stockholm-syndrome/index.html">Stockholm Syndrome</a></li>
+          <li><a href="/subpages/uncanny-valley/index.html">Uncanny Valley</a></li>
+          <li><a href="/subpages/observer-expectancy-effect/index.html">Observer-expectancy effect</a></li>
+          <li><a href="/subpages/base-rate-fallacy/index.html">Base Rate Fallacy</a></li>
```
Comment (mauriciopasquier) [inline subpages/about/index.html]: You missed all these /index.html

**Review by mauriciopasquier — CHANGES_REQUESTED** [review]

File: subpages/about/index.html (hunk 1 above)
Comment (mauriciopasquier) [reply subpages/about/index.html]: I think most of the href still use `/index.html`? I thought the idea was to remove them as well.

Comment (espimadrazo) [reply subpages/about/index.html]: Just wanted to see if you were paying attention :eyes:

Comment (espimadrazo) [reply subpages/about/index.html]: Done!

**Review by mauriciopasquier — APPROVED** [review]

**Comment by mauriciopasquier** [comment]
I didn't test it but it looks good!

**Comment by espimadrazo** [comment]
> I didn't test it but it looks good!

_"We didn't playtest this at all"_

**Pull request merged**